### ISSUE TITLE
Add ZKP comparison table

### ISSUE THREAD

**Issue by alxiong** [ZKProofs/ZKProofs.github.io#43]
Please add comparison table for major ZKP systems, something like:

| | Pinocchio | Aurora18 | Bulletproof | ... |
|-- | --- | --- | --- | ---|
| Prover | O (\|C\|) | ... |
| Verifier | O(\|x\|) | ... |
| Proof Size | O(1) 288 bytes | ... |
| Trusted setup | yes | ... |
| ... | ... |

The comparison dimensions may include:

- Quantitative: 
  - Prover time
  - Verifier time
  - Proof size
  - Setup/pre-processing time

- Qualitative: 
  - Public Verifiable (v.s. designated verifiable)
  - Trusted setup (v.s. transparent)
  - succinct (v.s. non-succinct, or even fully-succinct)
  - cryptographic assumptions (DLog, OWF, q-SDH, q-KEA etc.)
  - Argument system (v.s. proof system)
  - Interactive (v.s. NI)
  - for NP (v.s. maybe subset of NP, or v.s. potentially NEXP ?)

additional notes for quantitative measures: not only should the asymptotic complexity be specified, the concrete coefficients should also be included for some due to many recent works towards better practicality (e.g. while PHGR13 and Groth16 share the same O(1) constant proof size, Groth16 contains only 3 field elements whereas PHGR13 require 8)

To summarize, this request has the following to-dos:

- [ ] reach consensus whether we should have 2 comparison tables separately -- i.e. one for information-theoretical components (PCP, LIP, IOP based), and one for those crypto compilers (e.g. DARK)  OR should we aggregate them into one table
- [ ] reach consensus on the measurement dimensions/metrics 
- [ ] a markdown table 
- [ ] same table content, but in a Latex for academics for easy copy-paste into their evaluation section.
